### ISSUE TITLE
I hope this library could be part of msys2's library

### ISSUE THREAD

**Issue by asmwarrior** [Blake-Madden/Wisteria-Dataviz#25]
There are a lot of libraries which can be installed by msys2's pacman command, see:

https://packages.msys2.org/package/

But there are few ones which support wxWidgets. Hopefully this library will arrive there.

Thanks.

**Comment by Blake-Madden** [comment]
Thanks, I'll certainly check that out.

My experience with package management and deployment is pretty weak, so it might be a while before I come up with something though.

**Comment by Blake-Madden** [comment]
No, not yet. Sorry, I'm moving this issue to either a discussion or project here in a little bit, just trying to get things reorganized.
